src/queries.js

```diff
@@ -132,7 +133,17 @@ export default function queries(allQueries) {
               }
 
               // TODO(gio): support props renaming
-              this._subscription = this.context.avenger.queries(queries).subscribe(::this.setState);
+              this._subscription = this.context.avenger.queries(queries).subscribe(q => {
+                // add `ready` boolean param to readyState
+                const queriesProps = {
```

Comment:
bad name? replace with `queryState`?
1. I think `q` is a single `query`, not `queries`
2. here we're saving the query `state`, not its props (which are `params` and other `queries` it depends on)